apps/modernization-ui/src/pages/patient/profile/identification/identification.ts

```diff
@@ -6,3 +6,11 @@ export type IdentificationEntry = {
     state: string | null;
     sequence?: number;
 };
+
+export enum Headers {
```

Comment:
nit:  You may want to consider renaming this.  ES-lint has started complaining about `Headers` already being defined as a global import.  Would `Column` make sense?

Reply:
Column does make sense.